.github/workflows/add-release-reviewers.yml

```diff
@@ -9,29 +9,42 @@ permissions: {}
 
 jobs:
   add-reviewers:
-    if: ${{ github.event.label.name == 'release' }}
     # Tests that label is added on the PR
+    if: ${{ github.event.label.name == 'release' }}
     runs-on: ubuntu-latest
     permissions:
-      contents: read
+      contents: write
+      pull-requests: write
     steps:
       - id: get-members
-        uses: garnertb/get-team-members@0482f68c88601800f85145570ea93bf8fcfcf46f
-        with:
-          org: mui
-          team_slug: x
-          role: maintainer
-          token: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo ::setOutput name=data::$(\
+            gh api \
+              -H "Accept: application/vnd.github+json" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              /orgs/${{ORG}}/teams/${{TEAM_SLUG}}/members \
+              | jq 'reduce inputs as $i (.; . += $i)')
```

Comment:
`set output` has kind of been deprecated for a while now. https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
That's why I use 'setOutput' here! 😝

Reply:
Yeah... just saw the mention of the stdout section. Got it!

Reply:
Shouldn't it be `core.setOutput` then?

Reply:
nope ... it would still be `setOutput`, but I changed it to `>> $GITHUB_OUTPUT` now anyways! Thanks!